### PR TITLE
Add separate home page

### PR DESCRIPTION
**Pull request by inglesp**
We will move this to index.html once we are ready to launch the site.

### CODE REVIEW

File: content/home.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+PyCon UK is the annual gathering of the UK Python community and its friends
+from around the world.  This year, the conference is held at [Cardiff City
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+We&rsquo;ll be putting on a programme of talks, workshops, and other events
+aimed at the whole Python community, from web developers to hardware engineers,
+from professionals to hobbyists.  There will be an education programme for
+teachers, and an expanded research programme aimed at scientists and
+researchers.
```
Comment (snim2) [inline content/home.md:16]: can we change "aimed at scientists and researchers" to "aimed at researchers from the humanities, arts and sciences." please.

**Comment by ghickman** [comment]
@snim2 – sorry I didn't see your comment! I'll fix this on master.

File: content/home.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+body_class: home
+title: Home
+---
+
+PyCon UK is the annual gathering of the UK Python community and its friends
+from around the world.  This year, the conference is held at [Cardiff City
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+We&rsquo;ll be putting on a programme of talks, workshops, and other events
```
Comment (meshy) [inline content/home.md:12]: This is insanely picky of me, I realise, but this should arguably be `&apos;`, not `&rsquo;`.

Realistically, though, is it worth escaping all of these unicode chars?

Comment (ghickman) [inline content/home.md:12]: We have charset set to utf-8 in the base template so yea I think we just drop all the escape codes, unless there's something in wok that might cause issues @inglesp?

Comment (meshy) [inline content/home.md:12]: To be done in https://github.com/PyconUK/2016.pyconuk.org/issues/14

**Pull request merged**